app/src/main/java/com/android/sample/ui/authentication/Login.kt

```diff
@@ -73,14 +67,7 @@ fun LoginScreen(navigationActions: NavigationActions) {
                                     append("Link")
                                   }
                             },
-                        style =
-                            TextStyle(
-                                fontSize = 20.sp,
-                                fontFamily = FontFamily(Font(R.font.roboto)),
-                                fontWeight = FontWeight(700),
-                                color = PrimaryPurple,
-                                letterSpacing = 0.2.sp,
-                                textAlign = TextAlign.Center))
+                        style = TypographyBeatLink.headlineLarge)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                        style = MaterialTheme.typography.headlineLarge)
```